ragna/deploy/_ui/modal_configuration.py

```diff
@@ -155,13 +190,8 @@ def change_upload_files_label(self, mode="normal"):
         else:
             self.upload_files_label.object = "<b>Upload files</b> (required)"
 
-    async def model_section(self):
```

Comment:
See previous comment on why this has been split into two synchronous functions (`create_config` and `model_section`).

Note that the only reason it was async previously was to make a call to `get_components`, which has now been moved to `main_page.py`.